octoprint_mrbeam/iobeam/airfilter.py

```diff
@@ -337,6 +357,36 @@ def set_temperatures(
         if temperature4 is not None:
             self._temperature4 = temperature4
 
+    def set_connected(self, connected):
```

Comment:
Having used @property before, maybe we can use @setter here?

Reply:
Or maybe not because there's quite a bit of logic...

Reply:
But a setter can have logic, thats why there is a setter and not direct access, right?